src/main/java/com/cglx/Calculator.java

```diff
@@ -44,4 +45,8 @@ public int remainder(int a, int b)
         int result = a%b;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01:9990/coding_rules#rule_key=squid%3AS1488)